uwblueprint/planet-read
# Migrate Login endpoint to GQL

## Notion ticket link
[Migrate login endpoint to GraphQL](https://www.notion.so/uwblueprintexecs/Migrate-login-endpoint-to-GraphQL-9aee1f27206a48f1ba5b3dcd36512646)



## Implementation description
* implemented the resolver for the login GQL mutation 
* updated the frontend to use the GQL mutation instead of the REST API



## Steps to test
Either through http://localhost:5000/graphql or the frontend,
1. Log in with the planet read admin/user credentials
2. Log in with invalid credentials [should return an error]



## What should reviewers focus on?
* `backend/python/app/graphql/mutations/auth_mutation.py` 
* `frontend/src/components/auth/Login.tsx` (mostly copied from starter-code)


## Checklist
